resources/planwise/sql/facilities.sql

```diff
@@ -13,9 +13,11 @@ FROM facilities
 ORDER BY name;
 
 -- :snip criteria-snip
-/*~ (if (:types params) */
+/*~ (if (:types params) (if (empty? (:types params)) */
+  AND 1=0
```

Comment:
It was already working with `facilities.type_id IN (:v*:types)`, as `(:v*:types)` expanded to an empty list